cryptostore/config.py

```diff
@@ -5,10 +5,16 @@
 associated with this software.
 '''
 import asyncio
+import logging
 import os
+from datetime import datetime
+from urllib.parse import urlparse
 
+import boto3
```

Comment:
i dont want to add this as a forced requirement, you'll need to do like the files in the data connectors. See this example where its using boto without actually importing it.

https://github.com/bmoscon/cryptostore/blob/master/cryptostore/data/s3.py#L11

Reply:
Makes sense, I'll make the changes and commit.